docs/scripts.js

```diff
@@ -26,14 +26,19 @@ function isIPaddress(ipaddress) {
   return /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.test(
       ipaddress);
 }
+
+function isLocal() {
+  // eslint-disable-next-line no-unused-expressions
+ return window.location.hostname.substring(window.location.hostname.lastIndexOf('.', window.location.hostname.lastIndexOf('.')) + 1) === 'local';
+}
 // Get scripts & styles from:
 // `localhost` while developing
 // `unpkg.com` while demoing OR if a specific version is specified
 // These SHOULD be global since they are called by some examples
 var loadScript = function (source, ver) {
   var external = source.startsWith('http');
   var from =
-    !ver && (external || window.location.hostname === 'localhost' || isIPaddress(window.location.hostname))
+    !ver && (external || window.location.hostname === 'localhost' || isIPaddress(window.location.hostname) || isLocal())
```

Comment:
Please put all the logic in a single function, merged with the logic from isLocal

Reply:
I want future devs to understand what we check here by reading the function's name

Reply:
@maoznir I don't understand the comment this function just checks if this is coming from a local domain and this is the name of the function.

Reply:
I updated the comment to reflect that too